lymph/unilateral.py

```diff
@@ -148,6 +150,19 @@ def init_edges(self, graph):
                     self.lnl_edges.append(new_edge)
 
 
+    def init_parameters(self):
+        """Initializes the paramaters dictionary including all edges.
+        """
+        self.parameters = {}

```

Comment:
I think this should be a private attribute (meaning that it starts with an underscore) and the name could also be more telling. Maybe something like `_edge_lookup`? Because it doesn't actually hold the parameters.